config/initializers/apipie.rb

```diff
@@ -19,6 +19,7 @@
     :operatingsystem_families => Operatingsystem.families.join(", "),
     :providers => ComputeResource.providers.join(', '),
     :default_nic_type => InterfaceTypeMapper::DEFAULT_TYPE.humanized_name.downcase,
+    :template_kinds => TemplateKind.all.map(&:name).join(", ")
```

Comment:
During Rails initialisation without a DB, the table won't exist.  I think if the lookup was lazy and happened inside the lambda, it'd be fine.  Perhaps use a lambda here and call it within the other, or move the code so it's not in this hash?